### DISCUSSION TITLE
Tracking doesn't finish

### DISCUSSION THREAD

**Discussion by transkriptase** [talmolab/sleap#1634] [General]
Hello @talmo, 

I am running SLEAP on hpc of my institute and having troubles. One of them is my tracks not finish. I run tracking with same settings on limited number of frames with settings;    

      "sleap-track **video** --frames 200-600 -tracking.tracker flow -m centroid_folder -m centered_instance_folder    **.slp**" 

it is working but definetely not with whole video, even after hours of running tracking does not finish (Predicting %100 but job not finish). The video is running without any setting 

       "sleap-track **video**  -m centroid_folder -m centered_instance_folder    **.slp**"

But in this case I can not get track_names from .h5 files, and they are really important for me to connect track breaks via python. I am running sleap with 50 individuals and my video is not long. 3 minutes video with 13000 frames in total. Do you have any idea why this is happening and how can I have track result with track_names at the end?

My second problem is training on hpc system; sadly after the training the .slp files I got has no predictions to add as labels. Only a few individuals in a few frames are predicted. I run same files woth local GPU computer with 20 random frames options, it was pretty good and I ould add predicted training data as labels. How can I do it with hpc. The command I am using for training is really simple. 

    " sleap-train training_config(centroid).json   .slp    video"
    " sleap-train training_config(centered_instance).json   .slp    video"

Thank you so much for everything. I watchd Animal Behavior Online Seminer last month, and can not wait for 3D tracking options!
Good luck.

**Comment by Marielamend** [comment]
Hello @transkriptase!

To be able to best assist you, can you send your logs if any, and any other screenshots of your process that can help us diagnose the problem.

Best,
Mariela

**Comment by Marielamend** [comment]
Hello @transkriptase!

Sorry you are experiencing these issues... can you try the following steps:

1. Try running `sleap-track` without tracking options (i.e., just poses).
2. If this completes, then try running tracking by itself (without re-predicting poses) and check if that finishes.

Here are some CLI examples of how to do these steps separately:

- [Inference with multi-model pipelines (e.g., top-down)](https://sleap.ai/guides/cli.html#examples:~:text=2.%20Inference%20with%20multi%2Dmodel%20pipelines%20(e.g.%2C%20top%2Ddown)%3A)
- [Re-tracking without pose inference](https://sleap.ai/guides/cli.html#examples:~:text=6.%20Re%2Dtracking%20without%20pose%20inference%3A)

Let us know how this goes!,
Mariela
